CHANGELOG.md

```diff
@@ -39,6 +39,11 @@
 #### Changed
 - Upgraded to IDE Rider v3.0.28 (#166)
 
+### Multiplayer Use Cases
+
+#### Changed
+- Upgraded to Unity 2022.3.27f1 (#171)
+
```

Comment:
Experimental samples aren't added into the changelog, and only added when they're released to the Basic subdirectory.